tests/installation/bootloader_s390.pm

```diff
@@ -495,33 +153,18 @@ sub run() {
     eval {
         ###################################################################
         # connect to zVM, login to the guest
-        my $reconnect = exists get_var("DEBUG")->{"no get_to_yast"};
-        $r = $s3270->connect_and_login($reconnect);
-
-        $self->get_to_yast() unless $reconnect;
+        $r = $s3270->connect_and_login();
+        $self->get_to_yast();
```

Comment:
the whole DEBUG hash is gone after https://github.com/os-autoinst/os-autoinst/pull/340, so it can't stay as is. readding a $reconnect variable is easy though - once you have the rest